jobs/tcp_router/monit

```diff
@@ -1,7 +1,17 @@
+check process haproxy
+  with pidfile /var/vcap/sys/run/bpm/tcp_router/haproxy.pid
+  start program "/var/vcap/jobs/bpm/bin/bpm start tcp_router -p haproxy"
+  stop program "/var/vcap/jobs/bpm/bin/bpm stop tcp_router -p haproxy"
+  group vcap
+
 check process tcp_router
-  with pidfile /var/vcap/sys/run/tcp_router/tcp_router.pid
-  start program "/var/vcap/jobs/tcp_router/bin/tcp_router_ctl start"
-    as uid vcap and gid vcap
-  stop program "/var/vcap/jobs/tcp_router/bin/tcp_router_ctl stop"
-    as uid vcap and gid vcap
+  with pidfile /var/vcap/sys/run/bpm/tcp_router/tcp_router.pid
+  start program "/var/vcap/jobs/bpm/bin/bpm start tcp_router"
+  stop program "/var/vcap/jobs/bpm/bin/bpm stop tcp_router"
   group vcap
+  depends on haproxy
+
+check file haproxy_config
+  with path /var/vcap/data/tcp_router/config/haproxy.conf
+  if changed timestamp then exec "/bin/bash -c '/var/vcap/jobs/tcp_router/bin/killer &>/var/vcap/data/tcp_router/config/killer.out'"
```

Comment:
why not log to `/var/vcap/sys/log/tcp_router/config_reloader.log` to be consistent with log location?